### PR TITLE
Update affinity-designer.json

### PR DESCRIPTION
**Pull request by gautamkrishnar**
Removed all unwanted spaces

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata post coming soon

File: share/goodie/cheat_sheets/json/affinity-designer.json [hunk 1]
```diff
@@ -1,435 +1,428 @@
...
-        "Format",
-        "Select",
-        "Tools",
-        "Brush",
-        "View",
-        "Opacity",
-        "Zoom"
+  "id": "affinity_designer_cheat_sheet",
+  "name": "Affinity Designer",
+  "description": "Vector Editor",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/affinity-designer.json]: @gautamkrishnar We can make this a little more descriptive

File: share/goodie/cheat_sheets/json/affinity-designer.json [hunk 2]
```diff
@@ -1,435 +1,428 @@
...
+  "metadata": {
+    "sourceName": "Affinity Designer Cheatsheet",
+    "sourceUrl": "https://i.imgur.com/qY4k7xX.jpg"
+  },
+  "section_order": [
+    "File",
+    "Edit",
+    "Layer",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/affinity-designer.json]: @gautamkrishnar "Imgur" is better as source name sourcename

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/affinity-designer.json]: Ok. I will change it.

**Comment by gautamkrishnar** [comment]
@moollaza pls review this

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**